usaspending_api/etl/management/commands/load_submission.py

```diff
@@ -139,7 +139,10 @@ def handle(self, *args, **options):
         # Once all the files have been processed, run any global
         # cleanup/post-load tasks.
         # 1. Load subawards
-        load_subawards(submission_attributes, db_cursor)
+        try:
```

Comment:
I think we need a larger conversation about how the loader should behave on a hard fail. Nothing else is operating in a try/catch right now (mostly because that was causing a lot of uncaught data issues).

If we need to handle fails, does it make sense to percolate loader errors up to a higher entity that can catch them? I'd rather not wrap one part of the process in such a generic `try/catch`.